app/src/main/java/com/hisham/ctintegrationsample/core/LocalStorage.kt

```diff
@@ -40,6 +40,26 @@ class LocalStorage @Inject constructor(@ApplicationContext private val context:
         }
     }
 
+    fun saveCurrency(value: String) {
```

Comment:
Can you change it and make it like `palette` variable. I'm using there the variable setter and getter to update/retrieve the value

Reply:
Sure, just one issue as SharedPreferences.getString(key, defValue) is nullable. So Kotlin is still considering it to be nullable even if we passed in a default value. Would it be wise to use the `!!` in this case? Because set() requires a non nullable value, right?